libs/core/src/components/pds-toast/pds-toast.scss

```diff
@@ -0,0 +1,176 @@
+:host {
+  --animation-duration: 0.3s;
+  --animation-timing: cubic-bezier(0.4, 0, 0.2, 1);
+  --padding-inline: var(--pine-dimension-lg);
+  --padding-inline-desktop: var(--pine-dimension-2xl);
+  --sizing-height-default: 68px;
+  --sizing-min-width: calc(var(--sizing-total-width) - (var(--padding-inline) * 2));
+  --sizing-min-width-desktop: calc(var(--sizing-total-width) - (var(--padding-inline-desktop) * 2));
+  --sizing-total-width: 350px;
+
+  box-sizing: border-box;
+  display: block;
+  font: var(--pine-typography-body-medium);
+}
+
+.pds-toast {
+  align-items: center;
+  animation: pds-toast--animate-in var(--animation-duration) var(--animation-timing);
```

Comment:
~~How does the `--animate-in` work without it being wrapped in `var()`? Also, should it read `--animate-inline`?~~

Leaving this for others, the formatting is weird but the name is for the animation defined below. 

As a small suggestion, I wouldn't use the `--` in the name.

Reply:
pds-toast--animate-in is not a variable its the name assigned to the keyframes for the animation.